sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -44,7 +44,8 @@ public void resetParams() {
     
     public void handleInitResult(boolean val) {
     	setInitSuccess(val);
-    	session.onSecurityInitialized();
+		if (session != null)
+			session.onSecurityInitialized();

```

Comment:
We still need to onSecurityInitialized to let the session know the security library is ready to go.  The session being null is more of an edge case related to timing around the disconnect as the close method and resetParams have the ability to set the session to null.

https://github.com/smartdevicelink/sdl_android/blob/41a6698a2cbd7b3428fc3be55efe30fc21c3965a/sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java#L35